src/client/eppo-client.ts

```diff
@@ -1046,10 +1074,10 @@ export default class EppoClient {
     try {
       if (this.assignmentLogger) {
         this.assignmentLogger.logAssignment(event);
```

Comment:
Are you considering any more major changes to the SDK behavior, such as all events getting put into a queue and processed by another worker?

Reply:
yeah, we'll need some mechanism for batching and uploading events. I'm thinking a simple `setInterval` logic that uploads ever N seconds, no need to overcomplicate. Thoughts?